DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -892,7 +918,9 @@ if ($setupSecrets -eq 'y') {
 if ($sparkCreation -eq 'y') {
     Write-Host -ForegroundColor Green "Setting up ScriptActions... (6/16 steps)"
     Write-Host -ForegroundColor Green "Estimated time to complete: 2 mins"
-    Add-ScriptActions
+	if ($useDatabricks -eq 'n') {
```

Comment:
same lets put the estimated time to complete in the if statement

Reply:
done